### PR TITLE
Added formatting into web viewer.

### PR DESCRIPTION
**Pull request by SirVer**
- Added prettier configuration.
- Reformatted code.

### CODE REVIEW

File: web_viewer/client/control.ts [hunk 1]
```diff
@@ -53,13 +53,25 @@ export class FirstPersonController {
     this.phiDelta = 0;
 
     window.addEventListener(
-        'keydown', event => this.onKeyDown(<KeyboardEvent>event), false);
+      'keydown',
+      (event) => this.onKeyDown(<KeyboardEvent>event),
+      false
```
Comment (pdubroy) [inline web_viewer/client/control.ts]: Nit: not for this PR, but I think you can just leave the `false` out here (and at least in JS, that argument is typically omitted unless it's `true`).

File: web_viewer/client/package.json [hunk 2]
```diff
@@ -5,15 +5,23 @@
...
+    "prettier": "^1.10.2",
     "@types/dat-gui": "0.6.1",
     "@types/three": "0.0.27",
     "dat-gui": "0.5.0",
     "three": "0.83.0",
     "typescript": "^2.2"
+  },
+  "prettier": {
+    "arrow-parens": "always",
+    "bracket-spacing": false,
```
Comment (pdubroy) [inline web_viewer/client/package.json]: Why not `"bracket-spacing": true` here? That's the style we're now using internally.

**Comment by SirVer** [comment]
All done.

@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user SirVer. Merge queue now has a length of 1.

**Pull request merged**